packages/calcite-components/src/components/list/list.tsx

```diff
@@ -221,6 +221,17 @@ export class List
     SelectionMode
   > = "none";
 
+  /**
+   * Specifies the interaction mode of the component.
+   *
+   * `"interactive"` allows interaction styling and pointer changes on hover
+   *
+   * `"static"` does not allow interaction styling and pointer changes on hover
+   *
+   * Do not combine `interaction-mode=“static”` and `selection-appearance=“border”`
```

Comment:
+@geospatialem @DitwanP for editorial review. 📖👀

Reply:
@aPreciado88 Can you speak more to the "why" behind not using the props together? Maybe something we can also add to enhance the component's documentation page. I didn't see context in the PR's summary or in the console warning, so wanted to be sure I understand the "why" a bit more before providing a review. Did observe different double quotes, so recommend changing those in the interim.

```suggestion
   * Do not combine `interaction-mode="static"` and `selection-appearance="border"`
```

Reply:
The reason for the console warning and not using the props together is described in this [issue](https://github.com/Esri/calcite-design-system/issues/8634). Under Calcite Design Solution > selection-appearance=“border”.

Reply:
Summarizing our discussions via chat:
- We'll enhance the above to mention it is recommended to only add the `interaction-mode="static"` and `"selection-appearance="none"` together (_see suggestion below_)
- We'll provide more context via some images to the `list`'s doc page under a new "Recommendations" section cc: @ashetland @jcfranco 

```suggestion
   * The `"static"` value should only be used when `selectionAppearance` is `"none"`.
```

Reply:
The property should be `selection-mode`, not `selection-appearance`.

Reply:
Updated the PR with the requested changes. Thanks for catching that @ashetland!